Examples/Scotty-tvOSExample/View Controllers/MiddleViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
```

Comment:
Similar comment to above about extracting this into a shared Examples/Common folder and then making it a member of both tvOS and iOS targets (named `OtherViewController.swift` in the iOS example).